### ISSUE TITLE
Can We Run MIN in Private Window Directly From Command Line?

### ISSUE THREAD

**Issue by iFarhaan** [minbrowser/min#1238]
I've tried lot of combination but it didn't work. I don't know if there is any solution to this. Like eg. **chromium  -incognito https://example.com** and **firefox -private https://example.com**

**Comment by Syndamia** [comment]
No, right now there are only 3 (4) [command line arguments](https://github.com/minbrowser/min/blob/61429877560451ff7bd0c9928c918685ec70bdcd/main/main.js#L98): 
- no arguments: just opens min
- URL (e.g. `min https://https://github.com/minbrowser`): this opens the URL in a new tab
- Search (e.g. `min -s minbrowser`): this opens a new tab with the search term entered in your search engine
- File (works only on Windows ; e.g. `min index.html`): opens a local HTML file in a new tab

I've been thinking on improving the CLI functionality (#336, #1161) but I still need to learn more about electron and this app in general. My schedule is becoming increasingly busy so I don't know when I'll be able to work on it.
